utils/setStatus.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const statusCodes = require('http').STATUS_CODES
+    , isUndefined = require('./tools').isUndefined
+    , setStatus = () => {
+        return function (status) {
```

Comment:
in looking through this I am 99% sure that setStatus doesn't need to return a function that the return function can just be the body of setStatus. There isn't anything being enclosed by the outer function as defined here so it's not really needed.